features/bootstrap/FeatureContext.php

```diff
@@ -14,7 +14,7 @@
 
 define( 'EE', true );
 define( 'EE_VERSION', trim( file_get_contents( EE_ROOT . '/VERSION' ) ) );
-define( 'EE_CONF_ROOT', '/opt/easyengine' );
+define( 'EE_OPT_ROOT', '/opt/easyengine' );
```

Comment:
`EE_OPT_ROOT` should be `EE_ROOT_DIR` or `EE_HOME_DIR`.

`opt` is part of the default. should not be part of constant itself. A user can move EE_ROOT to any dir which is not `opt`

Reply:
We can go with `EE_ROOT_DIR`. @sagarnasit

Reply:
Updated with the changes.